contracts/ParameterStore.sol

```diff
@@ -17,48 +21,77 @@ contract ParameterStore is AccessController {
     /// example for _archToLevRatio: If each arch is worth 1000 lvUSD, set this to 1000
     uint256 internal _archToLevRatio = 1 ether; // meaning 1 arch is equal 1 lvUSD
 
-    constructor(address admin) AccessController(admin) {}
+    modifier onlyGovernor() {
+        require(hasRole(GOVERNOR_ROLE, msg.sender), "Caller is not Governor");
+        _;
+    }
 
-    function init(address treasuryAddress) external initializer onlyAdmin {
+    function init(address treasuryAddress, address admin) external {
+        require(_initlized == false, "Cant re init contract");
+        _grantRole(DEFAULT_ADMIN_ROLE, admin);
+        _grantRole(GOVERNOR_ROLE, admin);
         _treasuryAddress = treasuryAddress;
+        _initlized = true;
+    }
+
+    /// TODO : Move access control to a simple lib
+
+    function setGovernor(address newGovernor) external {
```

Comment:
I think we should rename this. If there is only 1 address per role, then "set" makes sense.

But for this we can have multiple addresses, so I think we should change it to: add/remove
so `addGovenor` & `removeGovenor` or `revokeGovenor` is fine too